app/models/foodbag.rb

```diff
@@ -0,0 +1,6 @@
+class Foodbag < ApplicationRecord
+  enum pickuptime: [:morning, :afternoon, :evening]
+  enum status: [:available, :reserved, :collected]
+  validates_presence_of :pickuptime, :status, :donor_id
```

Comment:
you don't need to validate the presence of donor_id since you are creating it on the user directly

Reply:
removed